docs/concepts/lifecycle.md

```diff
@@ -32,7 +32,7 @@ Registry Events
 
 The `RegistryEvent`s are fired after the mod instance construction. There are two: the `NewRegistry` event and the `Register` event. These events are fired synchronously during mod loading.
```

Comment:
This paragraph needs to be updated as `RegistryEvent.NewRegistry` does not exist anymore.